### DISCUSSION TITLE
How would I add two justified texts to a panel

### DISCUSSION THREAD

**Discussion by wasi-master** [Textualize/rich#1443] [Q&A] [answered]
Basically, I wanna make something like this

![example](https://i.imgur.com/h4qBG1y.png)

I thought about using panels but they don't seem to support multiple items

**Comment by willmcgugan** [comment, marked as answer]
You can create a Table.grid and right align the second column.

```python
from rich import print
from rich.panel import Panel
from rich.table import Table


table = Table.grid(expand=True)
table.add_column()
table.add_column(justify="right")
table.add_row("some text", "May 04, 2021")
print(Panel(table))
```

**Reply by wasi-master** [reply]
Thanks for the quick answer will, greatly appreciate it!
